nginx-autoinstall.sh

```diff
@@ -447,6 +451,14 @@ case $OPTION in
 			echo --add-module=/usr/local/src/nginx/modules/nginx-module-vts
 		)
 	fi
+	
+	if [[ $RTMP == 'y' ]]; then
+		git clone --quiet https://github.com/arut/nginx-rtmp-module.git /usr/local/src/nginx/modules/nginx-rtmp-module
```

Comment:
It would be better to use tags instead of the master branch :)

Reply:
Otherwise LGTM

Reply:
It will fail to build without the latest [commit to master](https://github.com/arut/nginx-rtmp-module/compare/v1.2.1...master).

Had to fork the repo and add that fix manually when i did this, but has since been added. :+1: